doc/devel/documenting_mpl.rst

```diff
@@ -88,16 +93,89 @@ errors.  To unset it, use
 
 You can use the ``O`` variable to set additional options:
 
-* ``make O=-j4 html`` runs a parallel build with 4 processes.
-* ``make O=-Dplot_formats=png:100 html`` saves figures in low resolution.
-* ``make O=-Dplot_gallery=0 html`` skips the gallery build.
+.. code-block:: sh
+    
+    #runs a parallel build with 4 processes.
+    make O=-j4 html 
+    
+    #saves figures in low resolution.
+    make O=-Dplot_formats=png:100 html 
+    
+    #builds the gallery without executing the scripts
+    make O=-Dplot_gallery=0 html 
+
+    #Multiple options can be combined using e.g.    
+    make O='-j4 -Dplot_gallery=0' html
+    
+Windows
+~~~~~~~
+   
+The documentation is build using the ``make.bat`` file. The options are set using 
+environment variables and variables can be set either in the 
+``make.bat`` file or set on the command line befor running ``make.bat``.
+
+Environment variables are set with
+
+.. code-block:: sh
+
+    #in cmd
+    set SPHINXOPTS=-W
```

Comment:
Can we use an example that is not the default value? IMHO that would make more sense.